### DISCUSSION TITLE
how to build linux binary from source?

### DISCUSSION THREAD

**Discussion by leonpano2006** [piotrulos/MSCModLoader#203] [Help me] [answered]
i would like to build linux binary for arm64 arm32 for cross comparision for performance test
i would like to know how to build that binary

**Comment by piotrulos** [comment, marked as answer]
There is no source for linux binaries, these binaries are provided by unity. 
And because MSC uses 10 year old version of unity there are no arm builds available for such old version

**Comment by leonpano2006** [comment]
> There is no source for linux binaries, these binaries are provided by unity. 
> 
> And because MSC uses 10 year old version of unity there are no arm builds available for such old version

Where did unity provided them?
And do you mean such binary are universal?(like it also works on other unity games uses same engine version?)

**Reply by piotrulos** [reply]
`\Editor\Data\PlaybackEngines` there.
